### PR TITLE
Hacktoberfest : Move documentation to Github

### PR DESCRIPTION
**Pull request by dheerajodha**
# Hacktoberfest: Move docs to GitHub

Move docs from wiki to GitHub.

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did

### CODE REVIEW

**Comment by jansohn** [comment]
Thanks for the PR! Can you please add the two missing release notes from:
https://github.com/jenkinsci/deploy-plugin/releases

File: README.md [hunk 1]
```diff
@@ -8,7 +8,133 @@
 Jenkins Deploy Plugin
 =========================
 
-This plugin permits Jenkins to Deploy into containers and application servers.
+This plugin permits Jenkins to Deploy into containers and application servers. It takes a war/ear file and deploys that to a running remote application server at the end of a build. The implementation is based on [Cargo](http://cargo.codehaus.org/). The list of currently supported containers include:
```
Comment (MarkEWaite) [inline README.md:11]: Update URL to the current home of the Cargo project
```suggestion
This plugin permits Jenkins to Deploy into containers and application servers. It takes a war/ear file and deploys that to a running remote application server at the end of a build. The implementation is based on [Cargo](https://codehaus-cargo.github.io/cargo/Home.html). The list of currently supported containers include:
```

**Review by MarkEWaite — APPROVED** [review]
One URL fix that would be nice to have before it is merged.

**Comment by MarkEWaite** [comment]
@jansohn could you label this pull request as "hacktoberfest" or as "hacktoberfest-accepted" so that @dheerajodha will be credited on the Hacktoberfest site for the merged pull request?

**Pull request merged**